sdk/ai/azure-ai-inference/azure/ai/inference/prompts/parsers.py

```diff
@@ -0,0 +1,129 @@
+"""
```

Comment:
Microsoft copyrights also?

Reply:
Sounds good, let's decide how we combine the `Adapted from` together with Microsoft copyright? -- especially we do modify the file by referencing Inference SDK